scripts/variantstore/wdl/extract/get_columns_for_import.py

```diff
@@ -0,0 +1,253 @@
+import requests
+import argparse
+
+from terra_notebook_utils import table
+import re
+
+
+maxNumSamples = 50
+
+
+def get_sample_sets(workspace_namespace, workspace_name):
+    response = requests.get('https://rawls.dsde-prod.broadinstitute.org/api/workspaces/{workspace_namespace}/{workspace_name}/entities?useCache=true')
+    sample_sets = response.sample_set
+    return sample_sets
+
+# do we want to start and just check for defaults first?
+#def get_column_names(workspace_id, workspace_name):
+
+
+def get_column_data(workspace_id):
+    # We need to identify 3 things
+    # 1. Sample id field
+    # 2. vcf column name
+        # 3. vcf index column name
+
+    # We only sample a certain number of columns
+    numSamples = 0
+    columnSamples = {}
+
+    table_name = "sample"
+    # list_rows is a generator and it uses paging behind the scenes to make
+    # this call much more efficient
+    for row in table.list_rows(table_name):
+        # the id field is special, so handle it like it is...
+        if f"{table_name}_id" in columnSamples:
+            existingList = columnSamples[f"{table_name}_id"]
+            existingList.append(row.name)
+        else:
+            newList = [row.name];
+            columnSamples[f"{table_name}_id"] = newList
+
+        # handle the more general attributes
+        for columnName in row.attributes:
+            if columnName in columnSamples:
+                existingList = columnSamples[columnName]
+                existingList.append(row.name)
+            else:
+                columnSamples[columnName] = [row.attributes[columnName]]
+
+        # done iterating columns
+        numSamples += 1
+        if numSamples == maxNumSamples:
+            break
+    return (columnSamples, numSamples)
+
+def get_column_values(columnSamples, numSamples):
+    # time to start gathering some potential rows.
+    # how many samples did we actually take?
+    numSampledRows = numSamples
+    cutoffPoint = numSampledRows * 0.95
+    print(f"Sampled {numSampledRows} rows total. Throwing away any under {cutoffPoint}")
+
+    # does the col has strings only
```

Comment:
Nit: I imagine this is where the check down on line 95 used to live?  If so, tis comment can be cleaned up